doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1724,9 +1724,55 @@ Ltac2 features a toplevel loop that can be used to evaluate expressions.
    together with its type. This command is pure in the sense that it does not
    modify the state of the proof, and in particular all side-effects are discarded.
 
+Compilation
+-----------
+
+Ltac2 definitions can be compiled to OCaml.  The compiled definitions
+will then run natively instead of using the interpreter.
+
+.. cmd:: Ltac2 Compile {* @qualid }
+
+   Compiles the named Ltac2 definitions and any other Ltac2 definitions they depend on
+   for use in the current Coq process.  The compiled code is used to execute the definitions
+   until the state reset (end of file, end of module, end of section).
+
+   Ltac2 code in an :ref:`antiquotation <term-antiquotations>` will
+   not be compiled. Nor will it be detected by dependency analysis,
```

Comment:
I'm unclear on the user impact of this limitation on antiquotations.  Is it something that can be handled later or a permanent limitation?  IIUC this doesn't refer to tactic definitions because that's a command, which cannot appear in an antiquotation.  If an antiquotation calls a tactic that's been compiled, it will use the compiled version, right?